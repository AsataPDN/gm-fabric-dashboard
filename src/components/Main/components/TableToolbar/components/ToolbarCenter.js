import styled from "styled-components";

const ToolbarCenter = styled.div`
  display: flex;
  flex: 1 1 33%;
  flex-direction: row;
  justify-content: center;
`;

export default ToolbarCenter;
