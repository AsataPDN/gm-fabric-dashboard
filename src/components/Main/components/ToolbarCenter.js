import styled from "styled-components";

const ToolbarCenter = styled.div`
  flex: 1 1 33%;
  display: flex;
  justify-content: center;
`;

export default ToolbarCenter;
