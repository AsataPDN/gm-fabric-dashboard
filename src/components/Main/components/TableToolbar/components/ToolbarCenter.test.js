import React from "react";
import { shallow } from "enzyme";
import ToolbarCenter from "./ToolbarCenter";

describe("ToolbarCenter", () => {
  it("matches snapshot", () => {
    const aToolbarCenter = shallow(<ToolbarCenter />);
    expect(aToolbarCenter).toMatchSnapshot();
  });
});
