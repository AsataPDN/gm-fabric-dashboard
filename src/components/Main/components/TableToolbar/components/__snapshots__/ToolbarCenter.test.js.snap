// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarCenter matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1 1 33%;
  -ms-flex: 1 1 33%;
  flex: 1 1 33%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
/>
`;
